src/me/StevenLawson/TotalFreedomMod/TFM_Util.java

```diff
@@ -129,6 +129,16 @@ public static void bcastMsg(String message)
     {
         TFM_Util.bcastMsg(message, null);
     }
+    
+    public static void announceMsg(String message, ChatColor color)
```

Comment:
So you can do "announceMsg("Hello World!");" and it will be like an announcement.